pkg/userserver/userserver.go

```diff
@@ -47,10 +47,10 @@ func (a AdminUserServer) getUser(id string) (hfv1.User, error) {
 
 }
 
-func (a AdminUserServer) SetupRoutes(r *mux.Router) {
-	r.HandleFunc("/a/user/list", a.ListFunc).Methods("GET")
-	r.HandleFunc("/a/user/{id}", a.GetFunc).Methods("GET")
-	r.HandleFunc("/a/user", a.UpdateFunc).Methods("PUT")
+func (u UserServer) SetupRoutes(r *mux.Router) {
+	r.HandleFunc("/u/user/list", u.ListFunc).Methods("GET")
```

Comment:
Is the `u` prefix really necessary here? We should be able to just make them top level.

Reply:
No, this should be /a/user/list, because we only want admins listing all users. I just typo'ed it.

Reply:
OK. I was confused because you said it was backwards compatible earlier and I was like hmm 🤣